doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -2,794 +2,859 @@
 
 .. _migration_2.6:
 
-Migration guide for |NCS| v2.6.0 (Working draft)
-################################################
+Migration guide for |NCS| v2.6.0
+################################
 
```

Comment:
I think we should add "Table of contents", just as release-notes and software maturity. But with depth 3.
.. contents::
   :local:
   :depth: 3

Then toggle could be removed and it would still be easy to navigate. But that is optional.